torch/jit/annotations.py

```diff
@@ -119,6 +120,11 @@ def get_type_line(source):
     type_lines = list(filter(lambda line: type_comment in line[1], lines))
 
     if len(type_lines) == 0:
+        type_pattern = re.compile('#[\t ]*type[\t ]*:')
+        wrong_type_lines = list(filter(lambda line: type_pattern.search(line[1]), lines))
+        if len(wrong_type_lines) > 0:
+            print("WARNING: the annotation prefix in line " + str(wrong_type_lines[0][0])
+                  + " is invalid. MyPy requires the exact prefix: '# type:'")
```

Comment:
Also let's not refer to MyPy directly and make people think it's secretly running under the hood

Reply:
on the other hand, wouldn't we annoy the heck out of our user if we don't explain her why exactly we don't support this syntax and raise an error instead? I don't mind removing the mypy comment if you think that's the best course of action here.

Reply:
Yea i would rather not reference MyPy directly, there are other python type checkers as well. It's a python standard not a mypy standard

Reply:
You can refer to the PEP or something lol